tests/core/training/converters/test_story_markdown_to_yaml_converter.py

```diff
@@ -1,15 +1,23 @@
 import os
 from pathlib import Path
-from typing import Text
-
+from typing import Iterator, Text
+import asyncio
 import pytest
+from sanic.request import Request
 
 from rasa.core.training.converters.story_markdown_to_yaml_converter import (
     StoryMarkdownToYamlConverter,
 )
 from rasa.shared.constants import LATEST_TRAINING_DATA_FORMAT_VERSION
 
 
+@pytest.yield_fixture(scope="session")
+def event_loop(request: Request) -> Iterator[asyncio.AbstractEventLoop]:
```

Comment:
why is this needed? If the scope is `session` it should go into `conftest` imo. If this just for the module, then the scope should be `module`